src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Arrays;
+
+public class Main {
+    public static void main(String[] args) {
+        final int Figures_Count = 6;
+        FigureSupplier supplier = new FigureSupplier();
+
+        Figure[] figures = new Figure[Figures_Count];
+        for (int i = 0; i < figures.length; i++) {
+            if (i <= (figures.length / 2)) {
+                figures[i] = supplier.getRandomFigure();
+            } else {
+                figures[i] = supplier.getDefaultFigure();
+            }
+        }
+        System.out.println(Arrays.toString(figures));
```

Comment:
In this homework you should also create interface Drawable and implement method 'draw' for each figure
![image](https://user-images.githubusercontent.com/117547096/212538677-61a79d71-7a2c-496f-ae1d-5e254ec0596d.png)

```suggestion
        for (int i = 0; i < figures.length; i++) {
            figures[i].draw();
        }
```